components/automate-chef-io/README.md

```diff
@@ -261,6 +261,22 @@ Block comments support markdown tables:
   rpc ListReports(Query) returns (Reports) {
 ```
 
+### Excluding an endpoint
+
+You can exclude an endpoint from the swagger generation by importing this in your proto file:
+
+```go
+import "protoc-gen-swagger/options/annotations.proto";
+```
+
+And adding this annotation to the rpc in question:
+
+```go
+option (grpc.gateway.protoc_gen_swagger.options.openapiv2_operation) = {
+  tags: "hidden";
+};
+```
```

Comment:
This is already mentioned in the docs, but got kind of buried: https://github.com/chef/automate/blob/ea926991bf9e335812713a7a8585cff3ae504bc9/components/automate-chef-io/README.md#notes